lld/ELF/Writer.cpp

```diff
@@ -1615,17 +1615,31 @@ template <class ELFT> void Writer<ELFT>::finalizeAddressDependentContent() {
       sec->addr = 0;
 
   // If addrExpr is set, the address may not be a multiple of the alignment.
```

Comment:
I think these comments would be clearer split up and moved adjacent to the checks within the loop. The loop now doesn't have an easy-to-describe purpose itself; it now exists just to service the checks within it.

Reply:
Moved!